tests/cache/models/samples/upsertBulk.js

```diff
@@ -358,20 +357,26 @@ describe('api::redisEnabled::POST::bulkUpsert ' + path, () => {
         },
       ])
       .then(() => {
-        api.get('/v1/samples?name=' +
-          `${tu.namePrefix}Subject|${tu.namePrefix}Aspect1`)
-        .set('Authorization', token)
-        .end((err, res) => {
-          if (err) {
-            done(err);
-          }
+        /*
+         * the bulk api is asynchronous. The delay is used to give sometime for
+         * the upsert operation to complete
```

Comment:
We have some thing similar in tests/jobQueue/v1/getBulkUpsertStatus.js and tests/jobQueue/v1/bulkUpsert.js. Here we are testing the bulkUpsert api without the workers.